packages/generators/src/addon-generator.ts

```diff
@@ -1,5 +1,6 @@
 import fs from 'fs';
 import path from 'path';
+import createLogger from 'yeoman-environment/lib/util/log';
```

Comment:
Let's avoid it,w e ahve built-in logger^^

Reply:
Oops